src/components/20-molecules/dropdown/index.js

```diff
@@ -271,15 +271,20 @@ class AXADropdown extends NoShadowDOM {
     }
 
     this.selectedIndex = integerIndex;
-    const [{ name }] = this.findByValue(value);
+    const [{ label }] = this.findByValue(value);
     // allow idiomatic event.target.value in onChange callback!
-    const syntheticEvent = { target: { value, index: integerIndex, name } };
+    const details = {
+      value,
+      name,
+      optionIndex: integerIndex,
+      optionLabel: label,
+    };
+    const syntheticEvent = { target: details };
```

Comment:
We are mixing the Option Object (https://www.w3schools.com/jsref/dom_obj_option.asp) with the Event Object of onChange. 
Why? To simulate the `target` at the syntheticEvent.

HTML Standard for `event.target.name` is the name of the select component. There is no "e.target.index" or "e.target.label". So i added them with the specific name "optionIndex". Its the index of the option, not the index of the select.

Reply:
I reverted the optionIndex part only (back to `index`), keeping the optionLabel. The reason is that we would like to remain backwards compatible and not break application code unnecessarily. Apps that relied on `index` do not have to be updated, since nothing changed semantically. optionLabel represents a meaningful change, but as a new key in the event object is not in danger of conflicting with existing app code.

Reply:
Dont like it.

Reply:
As discussed orally, we have now consensus that the revert of optionIndex => index is justified, since  it narrows the potential range of users who might suffer consequences of a breaking change.